### ISSUE TITLE
Panning on left mouse click-drag

### ISSUE THREAD

**Issue by sirrodgepodge** [vasturiano/3d-force-graph#317]
Hello there!

Thank you for an amazing library :).

I am right now attempting to use this for a 2D user interface (need to use this 3D lib to leverage WebGL for performance).

One thing which I have really tried to track down is how I might switch panning to happen with the left mouse click+drag instead of the right.  I have been able to turn off camera rotation like so:
`instance.controls().noRotate = true;`

For some reason however, it looks like this config object which `three.js` intends to expose in order to allow mouse behavior overrides is not available on `controls()` or `tbControls()` despite them both being instances of `TrackballControls` (setting this object to a value matching the config does not work either btw):
https://threejs.org/docs/#examples/en/controls/TrackballControls.mouseButtons

Any ideas?

**Comment by RaulPROP** [comment]
I'd also encountered this problem without being able to solve it.

In [this codepen](https://codepen.io/raulprop/pen/RwWgyLx?editors=0011) one can see that the buttons are re-arranged so the only action available should be a `pan` using the right button.

We can see [in the trackball code](https://github.com/mrdoob/three.js/blob/master/examples/js/controls/TrackballControls.js#L439) that if there is no button assigned it shouldn't do any aciton.

I also tried to log mouseButtons from `graph.controls()` but it printed nothing, so that made me thought that the object is created in an older version.

Checking the code of the nested libraries [I saw this](https://github.com/vasturiano/3d-force-graph/blob/eef3d2865835a886191b64c421987db8a4a80c97/src/3d-force-graph.js#L9)

I think that that library is deprecated and the `TrackballController` should be imported from `three.js`.

I will try to fork it and modify that import.
